server/data/data-jpa/src/test/java/org/apache/james/jpa/healthcheck/JPAHealthCheckTest.java

```diff
@@ -20,15 +22,13 @@
     void testWhenActive() {
         final JpaTestCluster JPA_TEST_CLUSTER = JpaTestCluster.create(JPAUrl.class);
         jpaHealthCheck = new JPAHealthCheck(JPA_TEST_CLUSTER.getEntityManagerFactory());
-        assertEquals(Result.healthy(jpaHealthCheck.componentName()),
-                jpaHealthCheck.check(), "Result should be healthy");
+        assertThat("Health check should be healthy", jpaHealthCheck.check(), is(equalTo(Result.healthy(jpaHealthCheck.componentName()))));
     }
 
     @Test
     void testWhenInactive() {
         jpaHealthCheck = new JPAHealthCheck((EntityManagerFactory)
                 new EntityManagerFactoryImpl().getBrokerFactory());
-        assertEquals(Result.unhealthy(jpaHealthCheck.componentName()),
-                jpaHealthCheck.check(), "Result should be unhealthy");
+        assertThat("Health check should be healthy", jpaHealthCheck.check(), is(equalTo(Result.unhealthy(jpaHealthCheck.componentName()))));
```

Comment:
sorry that's not the good assertThat. You are using hamcrest's assertThat instead of assertj's one...

Reply:
I'll fix this after work, just one question is the import order the default in Eclipse?

Reply:
yes!